functions/go/gatekeeper-validate/README.md

```diff
@@ -0,0 +1,43 @@
+# gatekeeper-validate
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Validate the KRM resources using the policy controller.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+You can use the policy controller to validate KRM resources. To learn more about
+the policy controller, see: https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller.
```

Comment:
Ditto, this is not the OSS project

Reply:
https://open-policy-agent.github.io/gatekeeper/website/docs/howto